app/ui.js

```diff
@@ -965,9 +983,171 @@ var UI;
 /* ------^-------
  *     /XVP
  * ==============
- *   CLIPBOARD
+ *    AST2100
  * ------v------*/
 
+        ast2100_handleVideoSettingsChanged: function (settings) {
+            if (!UI._ast2100_videoSettingsInitialized)
+                UI.ast2100_setDefaultSettings(settings);
+            UI.ast2100_updateVideoSettings(settings);
+        },
+        
+        // Called the first time we receive a FramebufferUpdate object.  Responsible
+        // for telling the server about any configured default settings.
+        ast2100_setDefaultSettings: function (settings) {
+            // Convert the settings that noVNC has been configured to set for all
+            // AST2100 servers to the familiar videoSettings format.
+            var defaultQuality = parseInt(UI.ast2100_quality);
+            var defaultSettings = {
+                quantTableSelectorLuma: defaultQuality,
+                quantTableSelectorChroma: defaultQuality,
+                subsamplingMode: parseInt(UI.ast2100_subsamplingMode)
```

Comment:
This seems like it might be a little bit beyond the scope of this PR.

One would have to produce an interface that abstracts both the ATEN settings (chrominance and luminance quantization tables each on an independent scale from 0 to 11 and a choice of two subsampling modes) and the TIGHT settings (a single "quality level" and another single "fine quality level", plus the choice of several more subsampling levels); then one would have to "un-hard-wire" the TIGHT settings; provide a UI that can sit on top of both; etc.